src/components/Nav/components/NavLinks/index.jsx

```diff
@@ -26,31 +26,20 @@ const GordonNavLinks = ({ onLinkClick }) => {
   };
 
   const dialogBox = () => {
-    if (dialog === 'offline') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Offline Mode'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable offline. Please reconnect to internet to access this feature.
-        </GordonDialogBox>
-      );
-    } else if (dialog === 'unauthorized') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Credentials Needed'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable while not logged in. Please log in to access it.
-        </GordonDialogBox>
-      );
-    }
+    const isOffline = dialog === 'offline';
+    return (
+      <GordonDialogBox
+        open={dialog}
+        onClose={() => setDialog(null)}
+        title={isOffline ? 'Unavailabile Offline' : 'Login Required'}
+        buttonClicked={() => setDialog(null)}
+        buttonName={'Okay'}
+      >
+        {isOffline
+          ? 'That page is not available offline. Please reconnect to internet to access this feature.'
+          : 'That page is only available to authenticated users. Please log in to access it.'}
+      </GordonDialogBox>
+    );
   };
```

Comment:
This wouldn't work because the `isOffline` condition is also checked on line 33 in the title prop. That's the reason I factored it out to a separate variable, so that we don't have to check it twice and risk introducing inconsistencies.